ncp.sh

```diff
@@ -131,8 +132,27 @@ EOF
 
   cat > /home/www/ncp-launcher.sh <<'EOF'
 #!/bin/bash
+
+declare -a args
+for arg in $@
```

Comment:
could use `optargs`, but this is also ok

Reply:
:thinking: 
Didn't know about optargs

Reply:
Removed in aaa4202d2bb8d9360a66c2b22703630ada44868c